org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,78 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: ghostwriter
+rename-icon: ghostwriter
+finish-args:
+- "--socket=wayland"
+- "--socket=x11"
+- "--socket=pulseaudio"
+- "--share=ipc"
+- "--share=network"
+- "--filesystem=home"
+- "--device=dri"
+- "--env=QTWEBENGINE_DISABLE_SANDBOX=1"
+modules:
+- name: cmark
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=Release"
+  - "-DCMAKE_INSTALL_PREFIX=/app"
+  sources:
+  - type: git
+    url: https://github.com/commonmark/cmark.git
+    tag: 0.30.3
+- name: cmark-gfm
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=Release"
+  - "-DCMAKE_INSTALL_PREFIX=/app"
+  sources:
+  - type: git
+    url: https://github.com/github/cmark.git
+    tag: 0.29.0.gfm.13
+- name: MultiMarkdown
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=Release"
+  - "-DCMAKE_INSTALL_PREFIX=/app"
+  sources:
+  - type: git
+    url: https://github.com/fletcher/MultiMarkdown-6.git
+    tag: 6.7.0
+- name: Pandoc
+  only-arches:
+  - x86_64
+  buildsystem: simple
+  build-commands:
+  - mkdir -p /app/
+  - cp -r bin/ /app/
+  sources:
+  - type: archive
+    only-arches: ["x86_64"]
+    url: https://github.com/jgm/pandoc/releases/download/3.1.6/pandoc-3.1.6-linux-amd64.tar.gz
+    sha256: 6eec0e0c85e97c90a02b9184c993d4bd848426fb3d7452ddc0d6014368c5e3fd
+  - type: archive
+    only-arches: ["aarch64"]
+    url: https://github.com/jgm/pandoc/releases/download/3.1.6/pandoc-3.1.6-linux-arm64.tar.gz
+    sha256: 6ca797f358b0cc04a3b9208ff1975a861bfa7110eafa881aa8c023ac4040e208
+- name: ghostwriter
+  buildsystem: cmake
+  builddir: true
+  build-options:
+    env:
+    - QMAKEPATH=/app/lib
+  config-opts:
+  - QMAKE_INCDIR+=/app/include/QtWebEngine
+  - QMAKE_INCDIR+=/app/include/QtWebEngineCore
+  - QMAKE_INCDIR+=/app/include/QtWebEngineWidgets
+  sources:
+  - type: git
+    url: https://github.com/KDE/ghostwriter.git
```

Comment:
aren't official KDE project pulled from tarballs from the KDE infra?

Reply:
Yes this should be https://download.kde.org/stable/release-service/23.04.3/src/ghostwriter-23.04.3.tar.xz